user/api.go

```diff
@@ -376,18 +367,12 @@ func (a *Api) UpdateUser(res http.ResponseWriter, req *http.Request, vars map[st
 				}
 			}
 
-			if updatedUser.EmailVerified && updatedUser.TermsAccepted != "" {
-				if a.marketoManager != nil && a.marketoManager.IsAvailable() {
-					if updateUserDetails.EmailVerified != nil || updateUserDetails.TermsAccepted != nil {
-						a.marketoManager.CreateListMembershipForUser(updatedUser)
-					} else {
-						a.marketoManager.UpdateListMembershipForUser(originalUser, updatedUser)
-					}
-				} else {
-					failedMarketoUploadCount.Inc()
-				}
-			}
 			a.logMetricForUser(updatedUser.Id, "userupdated", sessionToken, map[string]string{"server": strconv.FormatBool(tokenData.IsServer)})
+			if err := a.userEventsNotifier.NotifyUserUpdated(req.Context(), *originalUser, *updatedUser); err != nil {
+				a.logger.Println(http.StatusInternalServerError, err.Error())
```

Comment:
Can we add a prometheus counter for this error?

Reply:
done